scripts/globals/status.lua

```diff
@@ -813,8 +814,67 @@ tpz.effect =
     DYNAMIS                  = 800,
     MEDITATE                 = 801, -- Dummy effect for SAM Meditate JA
     ELEMENTALRES_DOWN        = 802, -- Elemental resistance down
-    -- PLACEHOLDER           = 803, -- Description
-    -- 803-1022
+    -- PLACEHOLDER              = 802, -- Description
+    -- 802-1022
+    INDI_REGEN               = 821,
```

Comment:
should even need any dummy effects to do this on the Geo. may -possibly- still need addStatusEffectEx to hide the icon if the loupan itself doesn't have one when targeted. The COLURE_ACTIVE status can hand out the status ID passed to it in its regular addStatusEffect() arguments (the only thing ex is adding is an argument for the icon separate from the actual status, allowing you to make any status say it is anything else-or- use zero for no icon).

Reply:
you are right, i dont need any of the effects now, they are all working off of the Colure now.